tests/phpunit/tests/auth.php

```diff
@@ -294,7 +294,7 @@ function test_empty_user_activation_key_fails_key_check() {
 	function test_legacy_user_activation_key_is_rejected() {
 		global $wpdb;
 
-		// A legacy user_activation_key is one without the `time()` prefix introduced in WordPress 4.3.
+		// A legacy user_activation_key is one without the `time()` prefix introduced in ClassicPress 4.3.
```

Comment:
ClassicPress 4.3 does not yet exists, this is legacy from WordPress, I think it should stay as is? Also L322